addon/components/denali-progress-enums.js

```diff
@@ -0,0 +1,22 @@
+/**
```

Comment:
Instead of creating a new file, can we create a `color-enums.js` file and share it between chip and progress components?

Reply:
created file and shared with chip and progress components